data/35e/sword_and_sorcery/campaign_setting/world_of_warcraft/core/wow_abilities.lst

```diff
@@ -1179,7 +1179,7 @@ Immunity to Non-Magical Disease	CATEGORY:Special Ability	TYPE:SpecialQuality
 Flight					CATEGORY:Special Ability	TYPE:SpecialQuality
 
 
-###Block: Dragons Redone Proposal	<Implement in the future - switch the the PF template system>
+###Block: Dragons Redone Proposal	<Implement in the future - switch the PF template system>
```

Comment:
That line is actually supposed to be "implement in the future - switch to the PF template system".